postgres_exporter/linux/crunchy-postgres-exporter@.service

```diff
@@ -15,7 +15,7 @@ After=network.target
 PermissionsStartOnly=true
 User=ccp_monitoring
 EnvironmentFile=/etc/sysconfig/%i
-ExecStartPre=/bin/bash -c "set -o pipefail; cat $QUERY_FILE_LIST | sed -e 's/#PG_STAT_STATEMENTS_THROTTLE_MINUTES#/${PG_STAT_STATEMENTS_THROTTLE_MINUTES}/g' -e 's/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES}/g' -e 's/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT}/g' > $$(echo $OPT | sed 's/.*--extend.query-path=\(.*\.yml\).*/\1/')"
+ExecStartPre=/bin/bash -c "set -o pipefail; cat $QUERY_FILE_LIST | sed -e 's/#PG_STAT_STATEMENTS_THROTTLE_MINUTES#/${PG_STAT_STATEMENTS_THROTTLE_MINUTES}/g' -e 's/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES}/g' -e 's/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT}/g' > $$(echo $OPT | sed 's/.*--extend.query-path=\([^\s]*\.yml\).*/\1/')"
```

Comment:
Testing just this sed with echo is not giving me correct restults:

Commands:
```
export OPT=" --web.listen-address=:9187 --extend.query-path=/etc/postgres_exporter/14/queries_golf_global.yml --disable-default-metrics --disable-settings-metrics --web.config.file=/var/lib/ccp_monitoring/postgres_exporter_web_config_file_golf_file.yml"
echo $OPT | sed 's/.*--extend.query-path=\([^\s]*\.yml\).*/\1/'
```

Results:
```
--web.listen-address=:9187 --extend.query-path=/etc/postgres_exporter/14/queries_golf_global.yml --disable-default-metrics --disable-settings-metrics --web.config.file=/var/lib/ccp_monitoring/postgres_exporter_web_config_file_golf_file.yml
```

Reply:
Weird that it works in the service file then. Any suggestions for a better fix?

Reply:
This seems to work?
```
echo $OPT | sed 's/.*--extend.query-path=\(.*\.yml\) .*/\1/'
/etc/postgres_exporter/14/queries_golf_global.yml```

Reply:
Did confirm that last change works just adding a space after the `(.*\.yml\)`

Least that seems to work consistently both in and out of the service file. Still open to better suggestions.

Reply:
Yogesh pointed out that last change doesn't work if --extend.query-path is the last option

Reply:
More testing seems to show that the below seems to work (replace the `\s` with my initial change with a plain space ` `)
```
sed 's/.*--extend.query-path=\([^\ ]*\.yml\).*/\1/'
```